### DISCUSSION TITLE
Calm iron golems with repairing

### DISCUSSION THREAD

**Discussion by Quik2007** [PurpurMC/Purpur#861] [__Completed__] [answered]
At the time you can calm a iron golem with a poppy, but wouldn’t it be nice if you could repair the golem completely to calm it down?

Thanks for your attention,
Quik2007

**Comment by BillyGalbreath** [comment, marked as answer]
Implemented in build 1531

https://github.com/PurpurMC/Purpur/commit/b7cc6931797a7c746ffd379f91649a18c2d45681
